pkg/plugins/policies/meshcircuitbreaker/plugin/xds/configurer.go

```diff
@@ -159,8 +159,13 @@ func configureSuccessRate(
 	}
 
 	if conf.StandardDeviationFactor != nil {
-		outlierDetection.SuccessRateStdevFactor = util_proto.UInt32(*conf.StandardDeviationFactor)
+		dec, err := v1alpha1.NewDecimalFromIntOrString(*conf.StandardDeviationFactor)
```

Comment:
Ideally we'd have a type in between here to avoid dealing with these impossible errors. Maybe we could even panic here? It has been validated after all...

Reply:
I don't know how could we introduce type in between because of https://github.com/kumahq/kuma/pull/5768#discussion_r1081463584

It's a good question whether we should panic or not after we validate in the first place.
In this PR I'd like to keep the err, but I'd love to resume a discussion in form of the issue, MADR or offline discussion within the team.